### PR TITLE
Add label for ClusterTask and image replacement for Controller

### PR DESCRIPTION
**Pull request by savitaashture**
# Changes

* Add image replacement for controllers, cluster tasks.
* Add label update `redhat` and `community` to clustertasks.

/cc @vdemeester @pradeepitm12 @nikhil-thomas 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/operator/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

```release-note
Add image replacement for controllers and clustertask
```

### CODE REVIEW

**Review by pradeepitm12 — APPROVED** [review]
/lgtm

File: pkg/reconciler/common/transformers.go [hunk 1]
```diff
@@ -18,12 +18,36 @@ package common
...
+	TriggersImagePrefix  = "IMAGE_TRIGGERS_"
+	AddonsImagePrefix    = "IMAGE_ADDONS_"
+
+	ArgPrefix   = "arg_"
+	ParamPrefix = "param_"
+
+	LabelProviderType     = "operator.tekton.dev/provider-type"
+	ProviderTypeCommunity = "community"
+	ProviderTypeRedHat    = "redhat"
+)
```
Comment (khrm) [inline pkg/reconciler/common/transformers.go:49]: I think we shouldn't use redhat or community terms here. Prolly it shoudl come from some flag or configmap.

Comment (khrm) [reply pkg/reconciler/common/transformers.go:49]: Or maybe we can include this logic in OpenShift.

Comment (vdemeester) [reply pkg/reconciler/common/transformers.go:49]: Looking at where it is used, I think we can define this in `openshift/*` rihgt ?

Comment (vdemeester) [reply pkg/reconciler/common/transformers.go:49]: Same for the other const.

**Review by vdemeester — COMMENTED** [review]
Except the const, looking good 👍🏼

File: pkg/reconciler/common/transformers.go (hunk 1 above)
Comment (savitaashture) [reply pkg/reconciler/common/transformers.go:41]: Updated PR

**Review by vdemeester — APPROVED** [review]
/cc @nikhil-thomas @pradeepitm12

**Comment by vdemeester** [comment]
Needs a rebase 😓

**Pull request merged**